test/protractor.conf.js

```diff
@@ -24,37 +24,24 @@ const config = {
       path: '../tools/axe-protractor/axe-protractor.js',
 
       rules: [
-        // Exclude md-menu elements because those are empty if not active.
-        { id: 'aria-required-children', selector: '*:not(md-menu)' },
+        // Exclude mat-menu elements because those are empty if not active.
+        { id: 'aria-required-children', selector: '*:not(mat-menu)' },
```

Comment:
You should just remove these rules for flex-layout. There is no test for `mat-menu` AFAIK.

Reply:
Yeah I think our only Material test dependency is on button. Either way, will remove